charts/operator-wandb/values.yaml

```diff
@@ -58,7 +58,10 @@ global:
     database: "wandb_local"
     user: "wandb"
     password: ""
-
+    passwordSecret:
+      name: ""
```

Comment:
What is name?  Why is this here?

Reply:
Ah, I see now.  `name` is the name of the Secret in the cluster, `key` is the identifier to grab within that secret.  We could do something like the following.  Right now it's all or nothing, if a user specifies a different secret they need to provide two passwords, otherwise the application password will have God access to the DB which is a big no no.

```
passwordSecret:
  name:
  rootKey: "MYSQL_ROOT_PASSWORD"
  userKey: "MYSQL_PASSWORD"

Reply:
Updated